orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -250,25 +254,62 @@ def _get_exported_model(self, *inputs, **kwargs):
         # correct training flag to reflect the expected behavior.
         # For example, the Dropout node in a model is dropped under eval mode.
         assert self._export_mode is not None, "Please use a concrete instance of ExecutionManager"
-
+        exported_model = None
         try:
-            with torch.no_grad(), _logger.suppress_os_stream_output(log_level=self._loglevel):
-                torch.onnx.export(self._flattened_module,
-                                  sample_inputs_as_tuple,
-                                  f,
-                                  input_names=self._input_info.names,
-                                  output_names=output_names,
-                                  opset_version=ONNX_OPSET_VERSION,
-                                  do_constant_folding=False,
-                                  training=self._export_mode,
-                                  dynamic_axes=self._input_info.dynamic_axes,
-                                  verbose=self._loglevel < _logger.LogLevel.WARNING,
-                                  export_params=False,
-                                  keep_initializers_as_inputs=True)
+            if self._onnx_export_type == torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH:
+                cloned_flattened_module = copy.deepcopy(self._flattened_module)
+                with _logger.suppress_os_stream_output(log_level=self._loglevel):
+                    torch.onnx.export(cloned_flattened_module,
+                                    sample_inputs_as_tuple,
+                                    f,
+                                    input_names=self._input_info.names,
+                                    output_names=output_names,
+                                    opset_version=ONNX_OPSET_VERSION,
+                                    do_constant_folding=False,
+                                    training=self._export_mode,
+                                    dynamic_axes=self._input_info.dynamic_axes,
+                                    operator_export_type=self._onnx_export_type,
+                                    custom_opsets={"prim": 1},
+                                    verbose=self._loglevel < _logger.LogLevel.WARNING)
```

Comment:
why not passing export_params=False and keep_initializers_as_inputs=True? This is important for large models (> 2GB)

Reply:
[peng]

Reply:
will follow exiting parameters for normal path.